.gitlab/pipelines/stages/deploy/stop_k8s.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/runtime/config.sh"
+
+sudo kill "$(cat "${K3S_PID_FILE}")"
```

Comment:
Would `killall ${K3S}` or whatever not work?

Reply:
`killall: command not found`

We will have to stick to the previous approach.

Reply:
I moved to ps | kill.